src/components/Comments.jsx

```diff
@@ -18,8 +18,12 @@ const Comments = () => {
         setIsError(true);
       });
   }, []);
+
+  if (comments.length === 0 && !isLoading) {
```

Comment:
remember to stick strictly to only pushing code relevant to this ticket on this branch